pkgs/pkgs-lib/tests/formats.nix

```diff
@@ -1005,4 +1005,51 @@ runBuildTests {
       </root>
     '';
   };
+
+  PlistGenerate = shouldPass {
+    format = formats.plist { };
+    input = {
```

Comment:
Could you add a bit more complexity to the test? lists containing items of multiple types, attrs of attrs, lists of lists, etc?